packages/plugin-default-event-tracking-advanced-browser/src/default-event-tracking-advanced-plugin.ts

```diff
@@ -108,19 +115,6 @@ export const defaultEventTrackingAdvancedPlugin = (options: Options = {}): Brows
     /* istanbul ignore next */
     const tag = element?.tagName?.toLowerCase?.();
 
-    // We only track these limited tags, as general text tag is not interactive and might contain sensitive information.
-    /* istanbul ignore if */
-    if (!DEFAULT_TAG_ALLOWLIST.includes(tag)) {
-      return false;
-    }
-
-    /* istanbul ignore if */
-    if (cssSelectorAllowlist) {
-      const hasMatchAnyAllowedSelector = cssSelectorAllowlist.some((selector) => element.matches(selector));
-      if (!hasMatchAnyAllowedSelector) {
-        return false;
-      }
-    }
```

Comment:
Curious, why was the cssSelectorAllowlist match check removed?

Reply:
The elements are already filtered out based on the `cssSelectorAllowlist`(https://github.com/amplitude/Amplitude-TypeScript/pull/606/files#diff-f0d884147fd15c876ca81736143fe16eff39e7d5547db18718c304519a7a9eabR195-R203). We need this before, because we filter the element by tags then by this filter.

Reply:
Thank you!